data/virt_autotest/guest_params_xml_files/opensuse_tumbleweed_64_xen_pv_x86_64.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0"?>
+<guest>
+    <guest_os_name>opensuse</guest_os_name>
+    <guest_version>tumbleweed</guest_version>
+    <guest_version_major></guest_version_major>
+    <guest_version_minor></guest_version_minor>
+    <guest_os_word_length>64</guest_os_word_length>
+    <guest_build></guest_build>
+    <host_hypervisor_uri></host_hypervisor_uri>
+    <host_virt_type>xen</host_virt_type>
+    <guest_virt_type>paravirt</guest_virt_type>
+    <guest_machine_type></guest_machine_type>
+    <guest_arch>x86_64</guest_arch>
+    <guest_name></guest_name>
+    <guest_domain_name></guest_domain_name>
+    <guest_memory>2048,maxmemory=4096</guest_memory>
+    <guest_vcpus>2,maxvcpus=4</guest_vcpus>
+    <guest_cpumodel></guest_cpumodel>
+    <guest_metadata></guest_metadata>
+    <guest_xpath></guest_xpath>
+    <guest_installation_automation>autoyast</guest_installation_automation>
+    <guest_installation_autoyast_file>opensuse_tumbleweed_xen_pv_guest_x86_64.xml</guest_installation_autoyast_file>
+    <guest_installation_method>location</guest_installation_method>
+    <guest_installation_extra_args>sshd=1#sshpassword=nots3cr3t#console=hvc0,115200n8#textmode=1</guest_installation_extra_args>
+    <guest_installation_wait></guest_installation_wait>
+    <guest_installation_method_others></guest_installation_method_others>
+    <guest_installation_media>http://openqa.qa2.suse.asia/assets/repo/openSUSE-Tumbleweed-DVD-x86_64-Snapshot12345</guest_installation_media>
```

Comment:
I don't think this path is accessible from the o3 network.

Reply:
Good catch! The link is for my testing in my own openqa server but I forgot to replace it before git push. Just changed it to "http://openqa.opensuse.org/assets/repo/openSUSE-Tumbleweed-DVD-x86_64-Snapshot12345"